core/chains/evm/gas/docs/FEE_HISTORY_ESTIMATOR.md

```diff
@@ -0,0 +1,56 @@
+# Fee History Estimator
+
+## Overview
+`Fee History` estimator is an EVM-based gas estimator that utilizes RPC calls to make gas price estimations. The estimator heavily relies on two RPC calls: `eth_gasPrice` and `eth_feeHistory`. It is built as a service and caches the calculated results in order to minimize overhead. It can bump past estimations by aggregating the result with the latest market prices. `Fee History` estimator supports both Legacy and Dynamic(EIP-1559) transactions. It can also handle chains that don't have a mempool and process transactions on an FCSF basis.
```

Comment:
nit:
```suggestion
`Fee History` estimator is an EVM-based gas estimator that utilizes RPC calls to make gas price estimations. The estimator heavily relies on two RPC calls: `eth_gasPrice` and `eth_feeHistory`. It is built as a service and caches the calculated results in order to minimize overhead. It can bump past estimations by aggregating the result with the latest market prices. `Fee History` estimator supports both Legacy and Dynamic(EIP-1559) transactions. It can also handle chains that don't have a mempool and process transactions on an FCFS basis.
```